README.md

```diff
@@ -31,9 +31,9 @@ Available Commands:
   completion      Modify your shell so doctl commands autocomplete with TAB
   compute         Display commands that manage infrastructure
   databases       Display commands that manage databases
-  help            Help about any command
+  help            Help with any command
   invoice         Display commands for retrieving invoices for your account
-  kubernetes      Displays commands to manage Kubernetes clusters and configurations
+  Kubernetes      Displays commands to manage Kubernetes clusters and configurations
```

Comment:
Here `kubernetes` is a `doctl` sub-command. So it should remain lower cased.

Reply:
Thank you for your response!  I have made changes as you suggested.